src/ontology/mondo-ingest.Makefile

```diff
@@ -256,7 +257,9 @@ $(REPORTDIR)/%_term_exclusions.txt $(REPORTDIR)/%_exclusion_reasons.robot.templa
 $(REPORTDIR)/%_exclusion_reasons.ttl: component-download-%.owl $(REPORTDIR)/%_exclusion_reasons.robot.template.tsv
 	$(ROBOT) template --input $(TMPDIR)/component-download-$*.owl.owl --add-prefixes config/context.json --template $(REPORTDIR)/$*_exclusion_reasons.robot.template.tsv --output $(REPORTDIR)/$*_exclusion_reasons.ttl
 
-$(REPORTDIR)/%_excluded_terms_in_mondo_xrefs.tsv $(REPORTDIR)/%_excluded_terms_in_mondo_xrefs_summary.tsv: $(TMPDIR)/mondo.sssom.tsv tmp/mondo.owl metadata/%.yml $(REPORTDIR)/component_signature-%.tsv $(REPORTDIR)/mirror_signature-%.tsv
+# todo: Should this also be a prereq $(TMPDIR)/component-download-$*.owl.owl? Perhaps worried about refreshing when not need to? But then we'd just use COMP=false if so?
```

Comment:
@joeflack4 is this still an active comment or has it been addressed?

Reply:
Active comment; it's unrelated to this PR, just something I noticed. Sometimes I commit these comments directly to `develop`, as is better if I have the time. Lowercase `todo`, indicating it is not critical.

Reply:
ok, but this is a question not an action item.

Reply:
Yes, basically the `todo` is to answer that question. I suppose almost anything with a `?` should be considered some temporary thing to be answered; not stay int he code. I find that adding `todo` makes this clearer; I can tell quickly which things are just plain explanatory comments, and which things need action. Part of that is because of IDE syntax highlighting.